README.md

```diff
@@ -51,11 +51,15 @@ Either assign superuser permissions to `postgres` or update the env vars: `GITHU
 
 ## Settings
 
-### Success Rates Department Settings
-
-- In order to change Department Success Rates time limit setting, create a new `Setting` with key prefix `success_rate`, followed by the department name and the metric name.
-- Example: `Setting.create!(key: 'success_rate_backend_merge_time', value: '12')`
-Possible values: 12 | 24 (default) | 36 | 48 | 60 | 72
+### Success Rates Settings
+- Department
+  - In order to change Department Success Rates time limit setting, create a new `Setting` with key prefix `success_rate`, followed by the department name and the metric name.
+  - Example: `Setting.create!(key: 'success_rate_backend_merge_time', value: '12')`
+  Possible values: 12 | 24 (default) | 36 | 48 | 60 | 72
+- Repository
+  - In order to change Repository Success Rates time limit setting, create a new `Setting` with key prefix `success_rate_repository`, followed by the metric name.
```

Comment:
what if we want to have different success rates in different repositories? shouldn't the key be `success_rate_[repo name]_merge_time` instead of `success_rate_repository_merge_time` ?

Reply:
Like it! Changed 😄